spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/source/SparkPositionDeltaWrite.java

```diff
@@ -104,6 +104,7 @@ class SparkPositionDeltaWrite implements DeltaWrite, RequiresDistributionAndOrde
   private final Context context;
 
   private boolean cleanupOnAbort = true;
+  private final Map<String, String> writeProperties;
```

Comment:
Minor: This variable should be co-located with other final variables above. It is super minor but let's do this in a follow-up.

Reply:
OK, I have raised a follow-up pr https://github.com/apache/iceberg/pull/8421.